websites/E/e-z/presence.ts

```diff
@@ -0,0 +1,115 @@
+const presence = new Presence({
+	clientId: "1018191690210754693",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://cdn.e-z.host/e-zimagehosting/ez256.png",
+		},
+		username = document.querySelector("p[id=premid-username]")?.textContent,
+		uid = document.querySelector("p[id=premid-uid]")?.textContent;
+
+	switch (document.location.hostname) {
+		case "e-z.host":
+			presenceData.buttons = [
+				{
+					label: "e-z.host",
+					url: "https://e-z.host",
+				},
+			];
```

Comment:
buttons to homepages are [not allowed](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets).
```suggestion
```